accesspoc/dips/views.py

```diff
@@ -14,9 +14,38 @@
 import zipfile
 
 
+def get_sort_params(request, options, default):
+    """
+    Get sort option and direction from request. Check options dict. for
+    available choices and use default if no option is passed on the request
+    or if the option is not valid. Defaults to asc. sort direction.
+    """
+    option = default
+    if 'sort' in request.GET and request.GET['sort'] in list(options.keys()):
+        option = request.GET['sort']
+
+    dir = 'asc'
```

Comment:
`dir` is a builtin function, you should probably use a different name, e.g. `dir_`.